.github/workflows/prod-check-package-version.yml

```
@@ -0,0 +1,26 @@
+name: 'Check package.json version'
+on:
+  pull_request:
+    branches:
+      - main
```

`pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```